### ISSUE TITLE
Unable to resize panes, create documents [Daily Build 2024.12.0+456, Win10]

### ISSUE THREAD

**Issue by GegznaV** [rstudio/rstudio#15456]
### System details

    RStudio Edition : Desktop [Open Source]
    RStudio Version : 2024.12.0+456 (daily build)
    OS Version      : Windows 10 x64 (build 19045)
    R Version       : R version 4.4.2 (2024-10-31 ucrt)

### Steps to reproduce the problem

A.

1. Install RStudio
2. Open RStudio, some documents were already present in Source Editor.
3. Try to resize the Console and Source Editor using these buttons:
![Image](https://github.com/user-attachments/assets/8c64b245-3490-45f0-a02f-a92543fe8156)
4. The size of the panes does not change.

B. 

1. Try to open a Quarto document using the following steps.
![Image](https://github.com/user-attachments/assets/5ddad557-60e2-49b0-b220-49cfee9334d1)
2. No response.

### Describe the problem in detail

It seems that RStudio UI malfunctions. Reinstalling the previous version solved the issue.

### Describe the behavior you expected

RStudio UI works as expected.

**Comment by ronblum** [comment]
I'm unable to reproduce this in RStudio Desktop 2024.12.0+456 on Windows 11. @GegznaV do you know what's the most recent version that worked for you?

**Comment by gtritchie** [comment]
@GegznaV Unfortunately we haven't been able to reproduce this, so it is difficult to take any action. Here are three things that might give us some clues:

1. Determine which version was last working. That would tell us which changes could have introduced the problem. All the builds are here: https://dailies.rstudio.com/rstudio/kousa-dogwood/electron/windows/.
2. Attach the diagnostic logs: first reproduce the problem, then `Help / Diagnostics / Show Log Files` and attach all the files in that folder (please double-check that they don't contain anything confidential). If `Show Log Files` is also not working, the files will be in `%localappdata%\rstudio\log`.
3. Check for Javascript messages/errors: start RStudio, then `Help / Diagnostics / Toggle Developer Tools` to show the Chromium debug tools. Make sure the debug Console is showing and reproduce the bug. Let us know what shows up in the devtools Console tab, if anything. If `Toggle Developer Tools` isn't working, you should be able to right-click anywhere in the RStudio interface and use "Inspect Element".

**Comment by GegznaV** [comment]
There were issues with the current released version 2024.09.1 as quarto templates did not work (always an empty document was inserted), so I tried to install daily which showed even bigger issues. Then I tried the previous released version, installation of which solved the described issues. Now the current release works well again.

I'll try to install the daily build to inspect if strange behavior persists.

Do I understand correctly that, if I already have RStudio, a new installation is not "totally clean" as some state files are kept?

**Comment by gtritchie** [comment]
> Do I understand correctly that, if I already have RStudio, a new installation is not "totally clean" as some state files are kept?

That is correct. Installing different versions of RStudio doesn't reset your settings, previous session state, etc.

https://support.posit.co/hc/en-us/articles/200534577-Resetting-RStudio-Desktop-s-State gives details on how you can reset some or all of this back to "factory settings".

There's also a batch file that can do this at https://github.com/rstudio/rstudio/blob/main/scripts/clean-rstudio.bat. For Mac and Linux, there's a shell script: https://github.com/rstudio/rstudio/blob/main/scripts/clean-rstudio.

**Comment by GegznaV** [comment]
UPDATE with the currently released version.

### System details

    RStudio Edition : Desktop [Open Source]
    RStudio Version : 2024.12.0.467
    OS Version      : Windows 10 x64 (build 19045)
    R Version       : R version 4.4.2 (2024-10-31 ucrt)

***

Using Control Panel, I tried creating a Quarto:

![Image](https://github.com/user-attachments/assets/e8ee34ec-3c8b-48ec-937c-4c0407139d84)

Result:
![Image](https://github.com/user-attachments/assets/1f19a06c-998f-41e6-86f8-cf18f5148122)

***

Resizing issues:

![Image](https://github.com/user-attachments/assets/959b4414-cafa-43b5-85ba-918020070d20)


***

Resize buttons behave in a strange way:

![Image](https://github.com/user-attachments/assets/ab8fe375-2e3d-40cf-8216-bc295db980df)

***
Buttons in some panes do not respond:

![Image](https://github.com/user-attachments/assets/df06d619-37be-4e9e-9465-7b94c213797f)

**Comment by GegznaV** [comment]
The issue repeats if I install RStudio with the antivirus disabled.

I noticed that the R startup message repeats twice:

![Image](https://github.com/user-attachments/assets/5f9e570a-b34c-4a3f-9d86-7cee106d2dfb)


----

Until now I tried these versions:

- RStudio Version : 2024.12.0.436 **issue is present**
- RStudio Version : 2024.11.0.331 **issue is present**
- RStudio Version : 2024.10.0.126 **issue is present**
- RStudio Version : 2024.9.0.375 **no issue**

**Comment by kevinushey** [comment]
On the off chance this is related to the rendering engine used by Electron under the hood, does the issue resolve if you switch that to Software? (In Tools -> Global Options... -> General -> Advanced)

<img width="677" alt="Image" src="https://github.com/user-attachments/assets/ed1af043-fff4-4cb3-ba05-6b80a97cf59a" />

**Comment by GegznaV** [comment]
Changing the rendering engine (I tried all 3 options) did not have any effect: resizing does not respond.

Disabling Electron accessibility support does not help either.

**Comment by gtritchie** [comment]
Have you tried resetting RStudio's state?

**Comment by gtritchie** [comment]
Note, this other similarly weird issue was resolved by resetting state: #15541

**Comment by gtritchie** [comment]
By chance do you have the "Enable publishing documents, apps, and APIs" setting turned **off** in Options / Publishing? 

There's a newly-discovered bug related to that causing various weird UI issues (#15561). This will be fixed in a patch release.

<img width="662" alt="Image" src="https://github.com/user-attachments/assets/bf104909-3bea-48ec-8f3c-4ec508fa5c00" />

**Comment by GegznaV** [comment]
> By chance do you have the "Enable publishing documents, apps, and APIs" setting turned **off** in Options / Publishing?

When the option is enabled, the issue is not present.

**Comment by gtritchie** [comment]
Ah, good to hear. At least now we know what's happening. A fix has been made and will be in an upcoming patch release. Thanks for your patience!

**Comment by GegznaV** [comment]
Is the fix present in the current daily release?

**Comment by gtritchie** [comment]
> Is the fix present in the current daily release?

A build hasn't been completed as of writing this. Check https://dailies.rstudio.com/rstudio/kousa-dogwood/ and watch for a newer build to appear sometime in next 15 hours or so. These builds are for an upcoming bug-fix release of the current 2024.12 release so should be reasonably stable (but use at your own risk given they are not official releases).

**Comment by gtritchie** [comment]
Marking this as fixed and ready for test; it is essentially a duplicate of https://github.com/rstudio/rstudio/issues/15561.

**Comment by gtritchie** [comment]
Actually, given the test case will be the same, I'm closing this and we'll track verification via #15561.

**Comment by aadler** [comment]
> By chance do you have the "Enable publishing documents, apps, and APIs" setting turned **off** in Options / Publishing?
> 
> There's a newly-discovered bug related to that causing various weird UI issues ([#15561](https://github.com/rstudio/rstudio/issues/15561)). This will be fixed in a patch release.
> 
> <img alt="Image" width="662" src="https://private-user-images.githubusercontent.com/10569626/399771430-bf104909-3bea-48ec-8f3c-4ec508fa5c00.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA2OTc1MjIsIm5iZiI6MTc0MDY5NzIyMiwicGF0aCI6Ii8xMDU2OTYyNi8zOTk3NzE0MzAtYmYxMDQ5MDktM2JlYS00OGVjLThmM2MtNGVjNTA4ZmE1YzAwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI3VDIzMDAyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1Y2U3NjE0YzliMTQyZjdkNTlkOTQ0MzQ0Mzc5ZTY3NTYxNTVjZGM1NmFlYjk2NWU3ODM1ZjMwZmIxZDVmNGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tSr9fW4ZsbQ9MzdBHCyFYRzGypW7EN9gue4se2sFmMY">

I had this issue with 2024.12.0-467 and checking the box seems to work.
